changelog.md

```diff
@@ -1,5 +1,15 @@
 # Changelog for [`hoopl` package](http://hackage.haskell.org/package/hoopl)
 
+## 3.10.1.1 *Apr 2015*
```

Comment:
I guess the `Apr 2015` date is a bit too optimistic, moreover the version here and in `hoopl.cabal` don't coincide...